frontend/fingertips-frontend/cypress/e2e/search.cy.ts

```diff
@@ -6,12 +6,16 @@ describe('Search Page', () => {
     const indicator = '123';
     Pages.searchPage.navigateToSearch();
 
+    // Assert
+    Pages.searchPage.checkA11Y();
```

Comment:
I worry that it would be easy to forget to check accessibility for a page, and easy to miss in review. Is there any way we could make the accessibility check run automatically when a page is loaded? I was going to suggest calling it from the "navigate" method, but we don't always use that to navigate to a page, do we?

Reply:
i could put it in the base functions navigateTo and checkURL - but then its kind of buried to whoever is looking at what the test is doing, worried doing this would make it harder to understand what was going on + debugging

Reply:
Yeah, that's a fair point too, and it breaks the single-responsibility principle for those methods if they then become responsible for checking accessibility as well as their primary purpose. Let's leave it as it is for now, as this test provides a template that shows the approach if we're implementing others.